tests/std/tests/VSO_0000000_c_math_functions/test.compile.pass.cpp

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#if BOTH

```

Comment:
These macros are either defined (to `1`) or not defined - we conventionally test such macros with `#ifdef` / `#if defined(MACRO)` to respect the off-by-default [warning C4668](https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-4-c4668?view=msvc-160) (that warns about Standard behavior, but which some people use in order to find `#if TYPOED_MACRO_NAME`). Although we aren't building this test with C4668 enabled, we should follow the convention.